extension/src/test/suite/Formatter.test.ts

```diff
@@ -225,6 +224,7 @@ suite("Lisp Formatter mock Tests", function () {
     // Test long list and big file - chinaMap.lsp
     // This test will take long time
     let i = 10;
+    this.timeout(20000);
```

Comment:
timeout?

Reply:
Since this file is a little big we should have more time for this single case to run the format.